javascript/src/main/pages/About/About.js

```diff
@@ -28,4 +29,4 @@ const About = () => {
   );
 };
 
-export default About;
+export default About;
```

Comment:
not a change that needs to be fixed, but in the future, please try to avoid committing files with only whitespace changes, as it clutters up the PR diff